src/vs/workbench/contrib/terminal/browser/media/CodeTabExpansion.psm1

```diff
@@ -0,0 +1,57 @@
+		[System.Management.Automation.CompletionResult]::new("-s", "-s", 'ParameterName', 'Print process usage and diagnostics information.'),
+		[System.Management.Automation.CompletionResult]::new("--status", "--status", 'ParameterName', 'Print process usage and diagnostics information.'),
+		[System.Management.Automation.CompletionResult]::new("--prof-startup", "--prof-startup", 'ParameterName', 'Run CPU profiler during startup.'),
+		[System.Management.Automation.CompletionResult]::new("--disable-extensions", "--disable-extensions", 'ParameterName', 'Disable all installed extensions. This option is not persisted and is effective only when the command opens a new window.'),
+		[System.Management.Automation.CompletionResult]::new("--disable-extension", "--disable-extension", 'ParameterName', 'Disable the provided extension. This option is not persisted and is effective only when the command opens a new window.'),
+		[System.Management.Automation.CompletionResult]::new("--sync", "--sync", 'ParameterName', 'Turn sync on or off.'),
+		[System.Management.Automation.CompletionResult]::new("--inspect-extensions", "--inspect-extensions", 'ParameterName', 'Allow debugging and profiling of extensions. Check the developer tools for the connection URI.'),
+		[System.Management.Automation.CompletionResult]::new("--inspect-brk-extensions", "--inspect-brk-extensions", 'ParameterName', 'Allow debugging and profiling of extensions with the extension host being paused after start. Check the developer tools for the connection URI.'),
+		[System.Management.Automation.CompletionResult]::new("--disable-lcd-text", "--disable-lcd-text", 'ParameterName', 'Disable LCD font rendering.'),
+		[System.Management.Automation.CompletionResult]::new("--disable-gpu", "--disable-gpu", 'ParameterName', 'Disable GPU hardware acceleration.'),
+		[System.Management.Automation.CompletionResult]::new("--disable-chromium-sandbox", "--disable-chromium-sandbox", 'ParameterName', 'Use this option only when there is requirement to launch the application as sudo user on Linux or when running as an elevated user in an applocker environment on Windows.'),
+		[System.Management.Automation.CompletionResult]::new("--telemetry", "--telemetry", 'ParameterName', 'Shows all telemetry events which VS code collects.'),
+		[System.Management.Automation.CompletionResult]::new("tunnel", "tunnel", 'Command', 'Make the current machine accessible from vscode.dev or other machines through a secure tunnel'),
+		[System.Management.Automation.CompletionResult]::new("serve-web", "serve-web", 'Command', 'Run a server that displays the editor UI in browsers.')
+	) | Sort-Object -Property CompletionText
```

Comment:
Why are you sorting here instead of just sorting the static list already?

Reply:
:+1: